### PR TITLE
[CHANGE] Update base image for tempo-query to 1.57.0

### LINKED ISSUE #3652: Can we upgrade the base image of tempo-query?
Even `grafana/tempo-query:latest` is still based on Alpine 3.16.7, I gather, and 3.16 is about to be EOL ([ref](https://alpinelinux.org/releases/)).

### PR DESCRIPTION
**Pull request by iblancasa**
**What this PR does**:
Bump `jaeger-query` image to 1.57.0.

**Which issue(s) this PR fixes**:
Fixes #3652

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 * [CHANGE] Update Alpine image version to 3.20 [#3710](https://github.com/grafana/tempo/pull/3710) (@joe-elliott)
 * [ENHANCEMENT] TraceQL - Add support for trace:id and span:id [#3670](https://github.com/grafana/tempo/pull/3670) (@ie-pham)
 * [BUGFIX] Fix TraceQL queries involving non boolean operations between statics and attributes. [#3698](https://github.com/grafana/tempo/pull/3698) (@joe-elliott)
+* [CHANGE] Bump Jaeger query docker image to 1.57.0 [#3652](https://github.com/grafana/tempo/issues/3652) (@iblancasa)
```
Comment (joe-elliott) [inline CHANGELOG.md:8]: this line needs to go above the # v2.5.0-rc.1 section

Comment (iblancasa) [reply CHANGELOG.md:8]: It is but I'll move to be just behind the title.

**Comment by kevincantu** [comment]
Thanks!  Anything else we need to do to get a review?

**Comment by kevincantu** [comment]
Thanks @iblancasa @joe-elliott!

**Review by electron0zero — APPROVED** [review]
lgtm, will wait for ✅ from @joe-elliott before I merge it.

**Review by joe-elliott — APPROVED** [review]

**Pull request merged**